files/CASSANDRA-9822/cassandra

```diff
@@ -91,7 +90,7 @@ do_start()
     start-stop-daemon -S -c cassandra -a /usr/sbin/cassandra -q -p "$PIDFILE" -t >/dev/null || return 1
 
     start-stop-daemon -S -c cassandra -a /usr/sbin/cassandra -b -p "$PIDFILE" -- \
-        -p "$PIDFILE" -H "$heap_dump_f" -E "$error_log_f" >/dev/null || return 2
+        -p "$PIDFILE" -E "$error_log_f" >/dev/null || return 2
```

Comment:
Do you not need to set this to the heap dump directory variable? Where is this new variable consumed?

Reply:
Ah this is explained in the description, I misread